### PR TITLE
Fixed: No longer require first run as admin on windows

### PR DESCRIPTION
**Pull request by bakerboy448**
(cherry picked from commit 8d9302da7761752189b2d2820915d7d13197ce47)
(cherry picked from commit daa9ee30a2c5fc2511e34167ce642c1bfe2a8b41)

#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ UpdateVersionNumber()
         echo "Updating Version Info"
         sed -i'' -e "s/<AssemblyVersion>[0-9.*]\+<\/AssemblyVersion>/<AssemblyVersion>$PROWLARRVERSION<\/AssemblyVersion>/g" src/Directory.Build.props
         sed -i'' -e "s/<AssemblyConfiguration>[\$()A-Za-z-]\+<\/AssemblyConfiguration>/<AssemblyConfiguration>${BUILD_SOURCEBRANCHNAME}<\/AssemblyConfiguration>/g" src/Directory.Build.props
-        sed -i'' -e "s/<string>10.0.0.0<\/string>/<string>$PROWLARRVERSION<\/string>/g" distribution/osx/Prowlarr.app/Contents/Info.plist
+        sed -i'' -e "s/<string>10.0.0.0<\/string>/<string>$PROWLARRVERSION<\/string>/g" macOS/Prowlarr.app/Contents/Info.plist
```
Comment (Qstick) [inline build.sh:24]: Keep this in distribution folder, this is the future structure, just haven’t changed the other repos

File: build.sh [hunk 2]
```diff
@@ -177,7 +177,7 @@ PackageMacOSApp()
 
     rm -rf $folder
     mkdir -p $folder
-    cp -r distribution/osx/Prowlarr.app $folder
+    cp -r macOS/Prowlarr.app $folder
```
Comment (Qstick) [inline build.sh:180]: Same

**Review by Qstick — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
Can't figure out / see why the installer create is failing....

**Comment by bakerboy448** [comment]
> > Can't figure out / see why the installer create is failing....
> 
> Did you read the error?

yes... but don't see where in the the build script it is missing the piece or I suppose missing/typoed folder.

```
Creating output directory: D:\a\1\s\distribution\windows\setup\output
[...]
Error on line 55 in D:\a\1\s\distribution\windows\setup\prowlarr.iss: Source file "D:\a\1\s\distribution\windows\setup\..\_artifacts\win-x64\net6.0\Prowlarr\Prowlarr.exe" does not exist.
Compile aborted.
cp: cannot stat 'distribution/windows/setup/output/Prowlarr.*win-x64.exe': No such file or directory
cp: cannot stat 'distribution/windows/setup/output/Prowlarr.*win-x86.exe': No such file or directory
```

**Comment by Qstick** [comment]
Baker: "We don't need this extra two levels"
image

Azure: 
`Source file "D:\a\1\s\distribution\windows\setup\..\_artifacts\win-x64\net6.0\Prowlarr\Prowlarr.exe" does not exist.`

Baker: "Why won't you work" :D

Just fix those path references since we have the `distribution\windows` folder here in Prowlarr

**Comment by bakerboy448** [comment]
baker: doh

thx

**Comment by Qstick** [comment]
np

File: azure-pipelines.yml [hunk 3]
```diff
@@ -164,6 +165,7 @@ stages:
           key: 'yarn | "$(osName)" | yarn.lock'
           restoreKeys: |
              yarn | "$(osName)"
+             yarn
```
Comment (Qstick) [inline azure-pipelines.yml:168]: purge this back out, it was removed to ensure no cross OS cache restore for yarn

File: azure-pipelines.yml [hunk 4]
```diff
@@ -816,6 +813,7 @@ stages:
...
```
Comment (Qstick) [inline azure-pipelines.yml:816]: and this

**Pull request merged**